### ISSUE TITLE
Portable Shaft Excavation Device does not use Phoron

### ISSUE THREAD

**Issue by Firecage** [Citadel-Station-13/Citadel-Station-13-RP#5888]
## Round ID:
[2158](https://scrubby.melonmesa.com/round/2158)

## Testmerges:
[#5876: 'allows pAIs to scan objects and mobs then display them as a hologram + add accessories to the pAI clothing options' by timothyteakettle at commit f7cc6257ee](https://github.com/Citadel-Station-13/Citadel-Station-13-RP/pull/5876)
[#5875: 'Fixes macros and micros getting Y-offsetted weirdly after unbuckling' by BlueWildrose at commit 8bcf5afa06](https://github.com/Citadel-Station-13/Citadel-Station-13-RP/pull/5875)
[#5867: 'Fixes hopefully all wide airlock misrotations in every map file' by BlueWildrose at commit 3f875136ee](https://github.com/Citadel-Station-13/Citadel-Station-13-RP/pull/5867)
[#5885: 'probably fixes rig modules' by silicons at commit d95a0fa52f](https://github.com/Citadel-Station-13/Citadel-Station-13-RP/pull/5885)

## Reproduction:
Basically, the Portable Shaft Excavation Device, that vertical phoron bore, does not use any Phoron despite having to be loaded with it. Albeit the power cell does still drain.

![image](https://github.com/Citadel-Station-13/Citadel-Station-13-RP/assets/4686269/5acf3c16-3cac-4f34-bef3-ebf9404f0c19)
![image](https://github.com/Citadel-Station-13/Citadel-Station-13-RP/assets/4686269/08e2f2b9-386b-408c-85b4-ffeab08b1ad0)
